src/shared/menu/helpers.ts

```diff
@@ -225,14 +225,15 @@ export function addIf(item: MenuItem, flag: boolean): MenuItem {
 /**
  * Helper function to create consistent menu entry doms
  * @param iconName The html of the svg to use as the icon
- * @param title The text to place in the button's title attribute
+ * @param content Either a string containing the text to place in the button's title attribute
+ * or an object containing the title and helpText to be used in the hover tooltip
  * @param key A unique identifier used for identifying the command to be executed on click
  * @param cssClasses extra CSS classes to be applied to this menu icon (optional)
  * @internal
  */
 export function makeMenuButton(
     iconName: string,
-    title: string,
+    content: string | { title: string; helpText: string },
```

Comment:
@b-kelly  I am not super happy with this change. I did not want to add a 5th argument to this function since the parameter list is already long enough. At the same time I did not want to change the signature for all function callers with an object (see the many callers in entries.ts file) so I settled with this union type.
Any suggestion?

Reply:
I think this is an ok change overall. I'll look at it in code and see if I can't think of a better way to express that without having to update every caller